types/google-map-react/tsconfig.json

```diff
@@ -16,7 +16,8 @@
         ],
         "types": [],
         "noEmit": true,
-        "forceConsistentCasingInFileNames": true
+        "forceConsistentCasingInFileNames": true,
+        "esModuleInterop": true
```

Comment:
This is generally a red flag, can you explain why you changed this setting?

Reply:
Yeah this was a strange one, felt a bit wrong but I got this error when the CI ran.

![image](https://user-images.githubusercontent.com/8209033/92476134-664f7480-f1d6-11ea-99da-ea650f7e1700.png)
Run link: https://github.com/DefinitelyTyped/DefinitelyTyped/commit/cf63b950170dcff67fda8d3950ef807da857eee8/checks?check_suite_id=1159255533

Honestly I don't understand this system of exports too well, never really used this syntax before. Any advice on this?